app/controllers/chargeA/ChargeDetailsController.scala

```diff
@@ -53,21 +51,14 @@ class ChargeDetailsController @Inject()(override val messagesApi: MessagesApi,
                                         val controllerComponents: MessagesControllerComponents,
                                         deleteChargeHelper: DeleteChargeHelper,
                                         config: FrontendAppConfig,
-                                        renderer: Renderer)(implicit ec: ExecutionContext)
+                                        view : ChargeDetailsView)(implicit ec: ExecutionContext)
     extends FrontendBaseController
     with I18nSupport
     with NunjucksSupport {
```

Comment:
sorry forgot to comment on this one
Please remove this `with NunjucksSupport` and import

Reply:
done